low-level/sspl-ll.spec

```diff
@@ -166,9 +162,11 @@ fi
 systemctl stop sspl-ll.service 2> /dev/null
 
 %postun
+SSPL_DIR=/opt/seagate/%{product_family}/sspl
```

Comment:
we have this defined above, why not reuse

Reply:
Done